appveyor.yml

```diff
@@ -1,6 +1,6 @@
 environment:
   password:
-    secure: uOcv5TOm/CgwOmiLQhNX+g==
+    secure: encrypted_pypi_password
```

Comment:
@kmike, I submitted a PR to crfsuite that will allow compiling crfsuite for python 2.6-3.5. In the meantime, we could release window wheels for all the python's except 3.5.

If you want to do that, you need to create an encrypted environmental variable https://ci.appveyor.com/tools/encrypt

like 

password=your_pypi_password

and put the whole string you get here: